### PR TITLE
Ignore unpressed events during velocity calculation on iOS

### PR DESCRIPTION
**Pull request by elijah-semyonov**
## Proposed Changes

Ignore unpressed events during velocity calculation on iOS.

## Testing

Test: check that no unintended fling happens during drag-and-stop sequences

### CODE REVIEW

File: compose/ui/ui/src/jvmMain/kotlin/androidx/compose/ui/input/pointer/util/VelocityTracker.jvm.kt [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.ui.input.pointer.util
+
+/**
+ * Some platforms (e.g. iOS) velocity calculation is incorrect when end position is used in velocity calculation.
+ */
+internal actual val onlyPressedEventsForVelocityTracker: Boolean
```
Comment (MatkovIvan) [inline compose/ui/ui/src/jvmMain/kotlin/androidx/compose/ui/input/pointer/util/VelocityTracker.jvm.kt]: Why do we need it in `jvmMain` source set? `notMobile` should cover it

Comment (elijah-semyonov) [reply compose/ui/ui/src/jvmMain/kotlin/androidx/compose/ui/input/pointer/util/VelocityTracker.jvm.kt]: We don't, indeed, it's a compilation error.

Comment (MatkovIvan) [reply compose/ui/ui/src/jvmMain/kotlin/androidx/compose/ui/input/pointer/util/VelocityTracker.jvm.kt]: But we need not forget to add actual to Android for upstreaming

Comment (elijah-semyonov) [reply compose/ui/ui/src/jvmMain/kotlin/androidx/compose/ui/input/pointer/util/VelocityTracker.jvm.kt]: Done

File: compose/ui/ui/src/commonMain/kotlin/androidx/compose/ui/input/pointer/util/VelocityTracker.kt [hunk 2]
```diff
@@ -285,6 +285,11 @@ private fun Array<DataPointAtTime?>.set(index: Int, time: Long, dataPoint: Float
     }
 }
 
+/**
+ * Some platforms (e.g. iOS) velocity calculation is incorrect when end position is used in velocity calculation.
+ */
+internal expect val onlyPressedEventsForVelocityTracker: Boolean
```
Comment (MatkovIvan) [inline compose/ui/ui/src/commonMain/kotlin/androidx/compose/ui/input/pointer/util/VelocityTracker.kt]: Suggestion: hide such details in ios-actual and use something like:
```kt
internal expect fun VelocityTracker.shouldUse(event: PointerInputChange)
```

Comment (elijah-semyonov) [reply compose/ui/ui/src/commonMain/kotlin/androidx/compose/ui/input/pointer/util/VelocityTracker.kt]: Sounds good.

**Pull request merged**